examples/src/main/java/com/mapbox/navigation/examples/activity/SimpleMapboxNavigationKt.kt

```diff
@@ -116,6 +124,29 @@ class SimpleMapboxNavigationKt : AppCompatActivity(), OnMapReadyCallback {
         }
     }
 
+    @SuppressLint("MissingPermission")
+    private fun initViews() {
+        bottomSheetBehavior = BottomSheetBehavior.from(bottom_sheet_faster_route)
+        bottomSheetBehavior.peekHeight = 0
+        faster_route_accept_progress.max = maxProgress.toInt()
```

Comment:
NIT Do we want to use snake notation or camel case? It seems we were using came case before (e.g. `startNavigation`). Don't have a strong preference but we should be consistent. Same thing applies to other views / layouts 👇